src/atomicui/organisms/UnauthSimulation/UnauthSimulation.tsx

```diff
@@ -165,8 +164,9 @@ const UnauthSimulation: FC<UnauthSimulationProps> = ({
 		bottomSheetHeight,
 		bottomSheetCurrentHeight = 0
 	} = useBottomSheet();
-	const { t } = useTranslation();
-	const currentLanguage = i18n.language;
+	const { t, i18n } = useTranslation();
+	const { language } = i18n;
+	const isLongLang = ["de", "es", "fr", "it", "pt-BR"].includes(language);
```

Comment:
Same here - shouldn't it be 'isTallLang'?

Reply:
resolved